### PR TITLE
dbus proof of concept / initial interface

### PR DESCRIPTION
**Pull request by dwlehman**
Future pull requests will add `Device` and `Format` interfaces with corresponding sub-trees.

After that will come removal of devices, initialization of disks, and a `Factory` interface to provide some high-level configuration of new device stacks.

### CODE REVIEW

**Comment by dwlehman** [comment]
I'm writing unit tests now.

**Comment by dwlehman** [comment]
Okay, I added some unit tests.

File: scripts/makebumpver [hunk 1]
```diff
@@ -232,12 +232,10 @@ class MakeBumpVer:
...
     def _rpmLog(self, fixedIn):
         git_range = "%s-%s-%s.." % (self.name, self.version, self.release)
-        proc = subprocess.Popen(['git', 'log', '--pretty=oneline', git_range],
+        proc = subprocess.Popen(['git', 'log', '--pretty=oneline', '--no-merges', git_range],
                                 stdout=subprocess.PIPE,
                                 stderr=subprocess.PIPE).communicate()
-        lines = filter(lambda x: x.find('l10n: ') != 41 and \
-                                 x.find('Merge commit') != 41 and \
-                                 x.find('Merge branch') != 41,
+        lines = filter(lambda x: x.find('l10n: ') != 41,
```
Comment (vpodzime) [inline scripts/makebumpver]: Do we still need even this filter? Isn't that from the times where we had translations in git?

Comment (dwlehman) [inline scripts/makebumpver]: Looks like it can go:
```
[david@rover blivet]$ git log | grep 10n
[david@rover blivet]$ 
```

**Comment by vpodzime** [comment]
Looks good to me otherwise.

**Comment by dwlehman** [comment]
Latest version removed the l10n filter from `makebumpver`.

I am seriously considering merging this onto `2.2-devel`, which would mean this pull request would never get merged.

**Comment by vpodzime** [comment]
> Latest version removed the l10n filter from makebumpver.

Looks good to me.

> I am seriously considering merging this onto 2.2-devel, which would mean this pull request would never get merged.

Works for me.

**Comment by dwlehman** [comment]
Closing this so I can open a new one after rebasing to `2.2-devel`.

**Pull request closed without merging**